docs/content/create-entity.md

```diff
@@ -11,37 +11,48 @@ To create an [entity](https://docs.improbable.io/reference/13.0/shared/glossary#
 
 ### 1. Create an entity definition using the `EntityBuilder`
 
-The `EntityBuilder` class is part of the `Improbable.Gdk.Legacy` assembly. It's a convenient way to create entity definitions, and allows you to specify [authority](authority.md) on a per-[component](https://docs.improbable.io/reference/13.0/shared/glossary#component) basis.
+The `EntityBuilder` class is part of the `Improbable.Gdk.Core` assembly. It's a convenient way to create entity definitions, and allows you to specify [authority](authority.md) on a per-[component](https://docs.improbable.io/reference/13.0/shared/glossary#component) basis.
 
```

Comment:
Comma after "definitions" reads oddly.

Are we pinned to 13.0? (Just checking.)

Reply:
We are pinned to a version that does not have docs on the website. Will change to latest as that's the closest to the version we have pinned.